user_guide_src/source/testing/feature.rst

```diff
@@ -379,7 +379,7 @@ Asserts that $fragment is found within the JSON response. It does not need to ma
     ]
 
     // Is true
-    $this->assertJSONFragment(['config' => ['key-a']);
+    $results->assertJSONFragment(['config' => ['key-a']]);
 
 .. note:: This simply uses PHPUnit's own `assertArraySubset() <https://phpunit.readthedocs.io/en/7.2/assertions.html#assertarraysubset>`_
```

Comment:
`assertArraySubset` is deprecated. I *think* we have already replaced this, but even if not let's just drop this note so as not to mislead.

Reply:
Yes. `assertJSONFragment` is no longer using phpunit's assertArraySubset. Better to drop that part.

Reply:
@MGatner done!